clouddatabasesv5/cloud_databases_v5_integration_test.go

```diff
@@ -388,6 +402,14 @@ var _ = Describe(`CloudDatabasesV5 Integration Tests`, func() {
 				MaxConnections: core.Int64Ptr(int64(200)),
 			}
 
+			if strings.Contains(deploymentID, "postgresql") || strings.Contains(deploymentID, "enterprisedb") {
```

Comment:
This test is already Postgresql specific and a `ConfigurationPgConfiguration` is already used before this. We should skip for PG or EnterpriseDB and address this in a different PR.